v03_pipeline/lib/reference_data/compare_globals.py

```diff
@@ -68,11 +69,10 @@ def from_ht(
         dataset_type: DatasetType,
     ):
         rdc_globals_struct = hl.eval(ht.globals)
-        paths = dict(rdc_globals_struct.paths)
-        versions = dict(rdc_globals_struct.versions)
+        paths = dict(rdc_globals_struct.get('paths', {}))
```

Comment:
added defaults here because some of the raw datasets that are used by crdqs don't necessarily have the standard paths/versions/enums globals

Reply:
Would it be better to just go through and fix them now?  I think we probably want to hard fail in the future if these fields are missing.

Reply:
good idea. I'll annotate globals for the existing crdq tables

Reply:
I removed the {} defaults but still need to update the most recent crdq tables before this is merged